### PR TITLE
adding Considerations for Keeping Images Up to Date conceptual

### LINKED ISSUE #456: How to keep your images up to date tutorial

### PR DESCRIPTION
**Pull request by SharpRake**
### What should this PR do?

resolves #456 

### What are the acceptance criteria? 

@amouat already reviewed a draft and I've implemented his suggestions, but if he wants to give it another look I wouldn't turn it down.

### How should this PR be tested?

This is a conceptual article, so no tech test. Just the regular content + formatting review, please.

### CODE REVIEW

**Review by imjasonh — COMMENTED** [review]
Lookin' good! Just a few comments.

I've written a bit about tags here too, if there's anything useful you can lift from it: https://docs.google.com/document/d/1Vbdm3T8ytQqiJuBHx6vCIzQaNEqGokZqFvEX--vHm5M/edit

File: content/software-security/secure-software-development/considerations-for-image-updates.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+There are many factors to consider when developing a process for keeping your images up to date, so you're unlikely to find useful advice on the subject beyond the general strategies outlined in this guide. Of course, different update strategies will work for different circumstances. Ultimately, whatever process you or your organization land on for keeping images up to date should suit the needs of your users without becoming a burden to you or your organization.
+
+
+## Learn more
+
+To reiterate, there's no one-size-fits-all approach to keeping one's images up to date. Our goal for this article is to introduce some of the important factors one should consider when developing a container image update plan for their application. If you'd like to learn more about the subjects touched on in this guide, we encourage you to check out the following resources.
+
+* [How to Use Chainguard Images](/chainguard/chainguard-images/how-to-use-chainguard-images/)
+* [How to Use Container Image DIgests to Improve Reproducibility](/chainguard/chainguard-images/videos/container-image-digests/)
```
[the remark this answers is not in the document]
Comment (SharpRake) [reply content/software-security/secure-software-development/considerations-for-image-updates.md:103]: oof, great catch!

File: content/software-security/secure-software-development/considerations-for-image-updates.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Before discussion image updates, it's helpful to have a baseline understanding of how images are typically versioned and named.
+
+[*Semantic versioning*](https://semver.org/) — also known as "semver" — is a system for determining how version numbers are assigned to a given piece of software. Software using semver has versions numbered in the format of `X.Y.Z`. `X` is reserved for major versions that are backwards incompatible, `Y` is used for minor versions that are backward compatible, and `Z` is used for patches and bug fixes. As an example, for a piece of software with the version number `3.5.2`, `3` is the major version, `5` is the minor version, and `2` is the patch number.
+
+Semantic versioning is intended to improve problems associated with having a large number of dependencies. Although semver has become a common practice throughout the software industry, it isn't used universally.
+
+When referencing an image, it's important to know exactly what image you are working with. Docker uses the following format for image names.
+
+```image
+[host[:port_number]]/[repository][image_name]
```
[the remark this answers is not in the document]
Comment (SharpRake) [reply content/software-security/secure-software-development/considerations-for-image-updates.md:34]: Ah yeah, that makes more sense.

**Review by imjasonh — APPROVED** [review]
🎉

**Review by jamonation — APPROVED** [review]
Looks like this needs another +1, so +1!

**Pull request merged**